modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -34,6 +40,28 @@ spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-updates'
 spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-uyuni-client-x86_64'
 ----
 
+endif::[]
+
+
+
+ifeval::[{uyuni-content} == true]
+
+.Procedure: Adding Channels and Repositories
+
+. At the command prompt on the {productname} Server, as root, install the [package]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the {centos} base, updates, and client channels using the [command]``centos6-uyuni-client`` script:
++
+----
+centos6-uyuni-client -u admin -p `secret` -a x86_64 'centos7'
+centos6-uyuni-client -u admin -p `secret` -a x86_64 'centos7-updates'
+----
```

Comment:
What you need is the `spacewalk-common-channels` script, and call it like this:
```
spacewalk-common-channels centos7 centos7-updates centos7-uyuni-client
```
For CentOS7.

And same for CentOS6, just replacing `7` with `6`